_includes/cohort-loop.html

```diff
@@ -0,0 +1,42 @@
+{% assign all_participants = '' %}
+{%- assign all_mentors = '' -%}
+{%- assign all_experts = '' -%}
+{%- assign all_facilitators = '' -%}
+{%- assign all_projects = '' -%}
+{%- assign cohorts = site.data.cohorts | sort %}
```

Comment:
Is it normal there is not `-` before `%}`?

Reply:
It is okay to leave it without `-` because the first one already handles whitespace. But for consistency, it might be best to put it back...